pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -151,15 +152,28 @@ type CephHealthMessage struct {
 	Message  string `json:"message"`
 }
 
-type ClusterState string
+type RookConditions struct {
```

Comment:
Can we simply name this `Conditions` instead?

Reply:
I already added a variable named `Condition` in the same file. That's why made the other one RookConditions; to avoid any confusion.

Reply:
Let's call this struct `Condition`. It should be singular and agreed that there is no need for the `Rook` prefix.

Reply:
Okay. I'll change it into `Condition`